website/blog/2019-08-31-dandelion-overview.md

```diff
@@ -1,73 +1,73 @@
+We expected this to be fairly straightforward. Turns out it's not. In order to make Dandelion Orgs work we dug into some of the lesser known features of the aragonOS like [ACL Oracles](https://hack.aragon.org/docs/acl_IACLOracle). ACL Oracles are small helper functions that plug in to Aragon's  access control list (ACL) to do more sophisticated permission evaluation. ACL Oracles are a complex and powerful feature of the aragonOS. In the future we plan to write a more thorough post about how we used them and how others might do the same. This post, however, will will be a high level overview of the apps that combine to make the Dandelion Org template.
 - Redemptions
 - Token Request
-- Lock
+- Time Lock
 - Delay
 - Dissent Oracle
 
-<img src='https://i.imgur.com/iWz0CR7.png' alt='Dandelion Org Overview' />
+These apps will be released individually for any Aragon DAO to use. We will also ship an easy to deploy template that combines all of these apps to create a Moloch like DAO on Aragon.
 
-We will release these apps individually for any Aragon DAO to incorporate as well as in an easy to deploy template that combines these apps to replicate the functionality of MolochDAO on Aragon.
+<img src='https://i.imgur.com/iWz0CR7.png' alt='Dandelion Org Overview' />
 
 ### Redemptions:
 >Allows users to manage a list of eligible assets held within an organization's Vault and allows members of the organization to redeem (burn) organization token in exchange for a proportional amount of the eligible assets.
 
-The Redemptions app by itself addresses the first of our requirements. It is similar in purpose to that of MolochDAO's ragequit function, but the logic for whether or not a member can redeem their token is externalized using Aragon's permission system.
+The Redemptions app is similar in purpose to that of MolochDAO's ragequit function. To accomplish this the logic of whether or not a member can redeem their token is externalized using Aragon's permission system.
 
-Also unlike the ragequit mechanism in MolochDAO, the Redemptions app works with both ETH and ERC20 tokens. An organization can manage an array of eligible tokens and users can redeem for a proportional share of all of these tokens.
+Unlike the ragequit mechanism in MolochDAO, Redemptions works with both ETH and ERC20 tokens. An organization can manage an array of eligible tokens for members to offer as tribute or to redeem from the vault. Like MolochDAO, members still receive a share of the DAO's assets proportional the amount of Dandelion Org tokens they are redeeming.
```

Comment:
`for members to offer as tribute` I don't think this should be here as it is not a function of redemptions.

Reply:
`Dandelion Org tokens` perhaps we should standardize on reputation rather than org tokens given that the assumption is that these tokens must be non-transferrable for the mechanism to work. Alternatively we could use "shares" to be conistent with moloch terminology.

Reply:
`Dandelion Org tokens` is specific enough that anyone reading knows that they're tokens of the Dandelion Org. `Reputation` and `shares` feels vague